src/drivers/boards/auav-x21/board_config.h

```diff
@@ -210,6 +210,9 @@
 
 #define BOARD_NAME "AUAV_X21"
 
+// Run the FMU task on its own thread to reduce jitter
+#define PX4FMU_TASK
```

Comment:
pretend for a minute that I actually enjoy talking about the C preprocessor, and explain to me how #1619 is relevant
Also, is there a style document detailing these preferences? It would perhaps help to avoid wasting time.